src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodTypeInference.cs

```diff
@@ -556,7 +556,7 @@ private void InferTypeArgsFirstPhase(Binder binder, ref HashSet<DiagnosticInfo>
             {
                 BoundExpression argument = _arguments[arg];
                 TypeWithAnnotations target = _formalParameterTypes[arg];
-                ExactOrBoundsKind kind = GetRefKind(arg).IsManagedReference() || target.Type.IsPointerType() ? ExactOrBoundsKind.Exact : ExactOrBoundsKind.LowerBound;
+                ExactOrBoundsKind kind = GetRefKind(arg).IsManagedReference() || target.Type.IsPointerOrFunctionPointer() ? ExactOrBoundsKind.Exact : ExactOrBoundsKind.LowerBound;
```

Comment:
>IsManagedReference [](start = 57, length = 18)

I suggest adding a PROTOTYPE comment here that the behavior for function pointers will have to be aligned with a specification once one is written.

Reply:
Added a comment here: [https://github.com/dotnet/roslyn/issues/39865](https://github.com/dotnet/roslyn/issues/39865)

---
In reply to: [406414229](https://github.com/dotnet/roslyn/pull/43041#discussion_r406414229) [](ancestors = 406414229)